src/modes/overview/vegaSpec.js

```diff
@@ -178,11 +178,12 @@ export function createSpec(sensor, selections, dateRange, valuePatch) {
             type: 'single',
             empty: 'none',
             on: 'mouseover',
-            nearest: false,
+            nearest: true,
+            encodings: ['x'],
```

Comment:
Sadly, it is exactly this addition of `encodings: ['x']` that interferes with showing the same gray line rule (and dot) in every chart.   This is the same change that makes the `nearest: true` setting be fast, since it avoids the voronoi computation, and it also stops the flickering of the tooltips, so it would be nice to be able to do this. 

I don't understand how all the other charts get this hover event, however.  Some magic is going on that I am not aware of.

Reply:
if you change `resolveHighlightedTimeValue` to

```js
export function resolveHighlightedTimeValue(e) {
  const highlighted = e.detail.value;
  if (highlighted && Array.isArray(highlighted.date_value) && highlighted.date_value.length > 0) {
    return Number.parseInt(formatAPITime(highlighted.date_value[0]), 10);
  }
  return null;
}

then it should consider the selection x only properly

Reply:
It does work with your resolveHighlightedTimeValue, thanks.  I was just starting to look at that function.  It does also work on the Region Details page, but not on the top-10 page.  I see there is a different onHighlight handler for the top-10 page so I'll see if I can make the appropriate changes there.

Reply:
I was able to fix the top-10 page as well.  And also fixed the warning about finding 'nearest' 'line' in the range selector by moving the highlight2 selection to the 'rule' layer.